app/api/routes.py

```diff
@@ -172,15 +172,16 @@ def get_gardenplant(id):
         if gardenplant.plant.harvest_time:
             harvest = gardenplant.harvest_date
             remaining = (harvest - today).days
+            formatted_harvest_date = gardenplant.harvest_date.strftime("%a, %B %d, %Y")
         else:
-            remaining = None
+            remaining = "N/A"
```

Comment:
Not sure why this was so brain bendy for us to figure out yesterday but glad its working now.  Turns out we were trying to calculate harvest date off of a Null Value.